batcher/aligned-sdk/src/core/constants.rs

```diff
@@ -8,6 +8,7 @@ pub const CONSTANT_GAS_COST: u128 =
         + BATCHER_SUBMISSION_BASE_GAS_COST;
 pub const DEFAULT_MAX_FEE_PER_PROOF: u128 =
     ADDITIONAL_SUBMISSION_GAS_COST_PER_PROOF * 100_000_000_000; // gas_price = 100 Gwei = 0.0000001 ether (high gas price)
+pub const CONNECTION_READ_TIMEOUT: u64 = 60; // 1 minute
```

Comment:
I don't think this protection is very strong. During the 60 seconds wait they could open at least a few hundred connections which in turn will also be granted 60 seconds of grace. Even with little distribution that could mean thousands of concurrent connections doing nothing.